test/plain_text_material

```diff
@@ -0,0 +1,4 @@
+It was the best of times, it was the worst of times,
```

Comment:
New choice of file name is worse than previous. Because it gives impression that it's suitable for *any* purpose, while it's not. Most notably at least it has to be short enough to be suitable for RSA processing. Unless of course it's moved to test/recipes/15-test_mp_rsa_data.